src/main/java/com/ust/sourcecourse/configuration/controller/ProjectGroupController.java

```diff
@@ -50,16 +70,85 @@ public ResponseEntity<List<ProjectGroupResponse>> getProjectGroupsByProjectUid(@
 		}
 	}
 
+	/**
+	 * 
+	 * @param id
+	 * @param update projectGroup
+	 * @return
+	 */
+
 	@PutMapping("/{id}")
 	public ResponseEntity<ProjectGroupResponse> updateProjectGroup(@PathVariable("id") Long id,
 			@RequestBody ProjectGroupRequest projectGroupRequest) {
 		ProjectGroupResponse updatedProjectGroup = projectGroupService.updateProjectGroup(id, projectGroupRequest);
 		return ResponseEntity.ok(updatedProjectGroup);
 	}
 
+	/**
+	 * 
+	 * @param id
+	 * @param delete projectGroup
+	 * @return
+	 */
 	@DeleteMapping("/{id}")
 	public ResponseEntity<Void> deleteProjectGroup(@PathVariable("id") Long id) {
 		projectGroupService.deleteProjectGroup(id);
 		return ResponseEntity.noContent().build();
 	}
+
+	/**
+	 * 
+	 * @param tags
+	 * @param add  tags to project Group
+	 * @return
+	 */
+	@PostMapping("/{id}/tags")
+    public ResponseEntity<ProjectGroup> addTagToProjectGroup(@PathVariable ("id") Long uid, @RequestBody  List<String> tags) {
+        ProjectGroup updatedProjectGroup = projectGroupService.addTagToProjectGroup(uid, tags);
+        return ResponseEntity.ok(updatedProjectGroup);
```

Comment:
Do not return project group entity class. Either return ProjectGroupResponse or return only the tags

Reply:
Resolved